src/components/Toast/Toast.tsx

```diff
@@ -0,0 +1,32 @@
+import { ReactNode } from "react"
+import { Status } from "./types"
+import styles from "./Toast.module.css"
+
+interface ToastProps {
+  status?: Status
+  message: string
+  action?: ReactNode
+}
+
+export function Toast({ action, message, status = "success" }: ToastProps) {
+  return (
+    <div
+      data-testid={`toast_${message}`}
+      role="alert"
+      aria-live={status === "error" ? "assertive" : "polite"}
+    >
+      <div
+        aria-label={`a ${status} toast`}
```

Comment:
Come to think of it, I'm not sure we need three separate divs to support the intended structure. 🤷

Reply:
You're right. I think this was a byproduct of refactoring.